rdt-open-reader/src/covidIndonesia/res/values-in/strings.xml

```diff
@@ -28,4 +28,7 @@
         <item>en</item>
         <item>in</item>
     </string-array>
+    <string name="lbl_back">Back</string>
+    <string name="lbl_gender_id">Female . ID: 99358357</string>
+    <string name="lbl_visit">Visit</string>
```

Comment:
Any reason to add these? They're not translated.

Reply:
removed.